### ISSUE TITLE
[BUG] clarinet test conflicting with recent deno std versions

### ISSUE THREAD

**Issue by hugocaillard** [hirosystems/clarinet#291]
### Issue

When requiring `https://deno.land/std@0.90.0/testing/asserts.ts` **(version >= 0.129)** in a test file, `clarinet test` throws the following error:
```
error: TS2352 [ERROR]: Conversion of type 'T' to type 'string' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
          ? diffstr(actual as string, expected as string)
                                      ~~~~~~~~~~~~~~~~~~
    at https://deno.land/std@0.129.0/testing/asserts.ts:357:39
```

### Step to reproduce

Simply import deno's assert in an empty test file:
```ts
import { assertEquals } from 'https://deno.land/std@0.90.0/testing/asserts.ts';
```

Run `clarinet test`
  
### Workaround
 
Downgrade to std@0.128 or lower

**Comment by moodmosaic** [comment]
Confirmed also on my local.  In my case, I can use up to [std@0.126.0](https://deno.land/std@0.126.0/testing/asserts.ts). Here's a repro, note that this uses [clarinet@v0.27](https://deno.land/x/clarinet@v0.27.0/index.ts):

```typescript
import { Clarinet, Tx, Chain, Account, types }
  from 'https://deno.land/x/clarinet@v0.27.0/index.ts';

import { assertEquals }
  from 'https://deno.land/std@0.126.0/testing/asserts.ts';

Clarinet.test({
    name: "Ensure that <...>",
    async fn(chain: Chain, accounts: Map<string, Account>) {
        let block = chain.mineBlock([
            /* 
             * Add transactions with: 
             * Tx.contractCall(...)
            */
        ]);
        assertEquals(block.receipts.length, 0);
        assertEquals(block.height, 2);

        block = chain.mineBlock([
            /* 
             * Add transactions with: 
             * Tx.contractCall(...)
            */
        ]);
        assertEquals(block.receipts.length, 0);
        assertEquals(block.height, 3);
    },
});
```

Output:
```
* Ensure that <...> ... ok (326ms)
```

Now, if I bump the version, to > std@0.126.0, e.g. std@0.127.0

```diff
diff --git a/foo_test.ts b/foo_test.ts
index b501bb7..942ad73 100644
--- a/foo_test.ts
+++ b/foo_test.ts
@@ -2,7 +2,7 @@ import { Clarinet, Tx, Chain, Account, types }
   from 'https://deno.land/x/clarinet@v0.27.0/index.ts';

 import { assertEquals }
-  from 'https://deno.land/std@0.126.0/testing/asserts.ts';
+  from 'https://deno.land/std@0.127.0/testing/asserts.ts';

 Clarinet.test({
     name: "Ensure that <...>",
```

Output:

```
Uncaught SyntaxError: Unexpected identifier
  override name = "AssertionError";
           ~~~~
    at <anonymous> (https://deno.land/std@0.127.0/testing/asserts.ts:20:12)
```

**Comment by lgalabru** [comment]
To be addressed with https://github.com/hirosystems/clarinet/pull/511

**Comment by hugocaillard** [comment]
Fixed with 1.0 and #511 👍 
Closing since the release is on the way
